package.json

```diff
@@ -32,9 +32,10 @@
     "editorconfig": "^1.0.1"
   },
   "lint-staged": {
-    "src/**/*.{html,css}": [
+    "src/**/*.{html,scss}": [
       "prettier --write --ignore-unknown",
       "eslint --fix --max-warnings=0"
     ]
-  }
+  },
+  "eslintIgnore" : ["src/**/*.scss"]
```

Comment:
Why do you use it here? `eslint --fix --max-warnings=0` no longer works???

Reply:
> Why do you use it here? `eslint --fix --max-warnings=0` no longer works???

i replied in slack.